playbooks/common/openshift-master/config.yml

```diff
@@ -5,6 +5,10 @@
     t_oo_option_master_debug_level: "{{ lookup('oo_option', 'openshift_master_debug_level') }}"
 
   pre_tasks:
+  - name: Fail early if Atomic and old version of Docker
+    include: ../../../roles/openshift_common/tasks/atomic_docker_check.yml
```

Comment:
I believe the main issue right now is that the variable needed is defined through openshift_facts, which I wouldn't have expected to be available at that point in the playbook. It turns out that we are running openshift_facts very early in the process by including `playbooks/common/initialize_facts.yml` from `playbooks/common/std_include.yml`, which is included from `playbooks/byo/openshift-cluster/config.yml`.

As for a better place to put the check without creating a new role, I'm not sure there is a good answer. Some of the least worst places I can think of:
- Putting the test in a task/playbook include that is pulled in after facts are initialized in `playbooks/common/openshift-cluster/std_include.yml`
- Putting the test directly in `roles/openshift_facts/tasks/main.yml`
- ???